### DISCUSSION TITLE
What happens to CML when its life remaining is 0?

### DISCUSSION THREAD

**Discussion by mnimu** [tearust/teaproject#44] [Q&A] [answered]
What happens to CML when its life remaining is 0?

**Comment by geopor** [comment, marked as answer]
Every CML has a lifespan of a few years. The epochs are accelerated during the contest, that's why you see the CML life measured in a number of days.

When a CML has 0 life remaining, its productive life is over and it's no longer useful for generating TEA rewards.

**Comment by kevingzhang** [comment]
the contest is running in so-called "fast mode". Everything is running at 100x speed.

Then a CML has zero life remaining, it is "dead". It will no longer generate TEA. You can unplant it without any penalty. Even if you do not unplant it, it will be removed in the next recycle window (usually every 1000 blocks). All stake slots on this CML will be unstake. 

If you have 1000T when plant, it will be refund to you. so does the debt. 

You will need to buy new CML seeds to plant new CML.
